xpublish/plugins/manage.py

```diff
@@ -0,0 +1,45 @@
+"""
+Load and configure Xpublish plugins from entry point group `xpublish.plugin`
+"""
+from importlib.metadata import entry_points
+from typing import Dict, Iterable, Optional
+
+from .hooks import Plugin
+
+
+def find_default_plugins(exclude_plugins: Optional[Iterable[str]] = None):
```

Comment:
```suggestion
def find_default_plugins(exclude_plugins: Optional[Iterable[str]] = None) -> dict[str, Plugin]:
```

Is this right?

Reply:
Good catch. I had a nagging sensation that I actually needed a `Type[X]` in some project I was working on but I couldn't remember where!

Reply:
Yep, it will be.